test/libsolidity/semanticTests/externalContracts/_prbmath/PRBMathCommon.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity >=0.8.0;
 
 /// @dev Common mathematical functions used in both PRBMathSD59x18 and PRBMathUD60x18. Note that this shared library
 /// does not always assume the signed 59.18-decimal fixed-point or the unsigned 60.18-decimal fixed-point
-// representation. When it does not, it is annonated in the function's NatSpec documentation.
+// representation. When it does not, it is annotated in the function's NatSpec documentation.
```

Comment:
Does this exist in the [upstream repo](https://github.com/paulrberg/prb-math) as well? If it does, would be nice to submit a correction. If it does not, we could just update the file.

Reply:
I checked their repo when I fixed it (though not for this particular one) and found nothing to be fixed

Reply:
The readme for this folder says the upstream is https://github.com/hifi-finance/prb-math/commit/62021c1abc3413f20d0bdc8f941cf9f21d5a7d2d though..